.github/workflows/build-binaries.yml

```diff
@@ -87,61 +38,18 @@ jobs:
 
       - name: Package binary
         run: |
-          mkdir dungeon-revealer-linux
-          mv ./bin/dungeon-revealer-linux ./dungeon-revealer-linux/
-          mv ./node_sqlite3.node ./dungeon-revealer-linux/
-          mv ./LICENSE ./dungeon-revealer-linux/
-          mv ./README.md ./dungeon-revealer-linux/
+          mkdir dungeon-revealer-${{ matrix.os }}
+          mv ./bin/dungeon-revealer* ./dungeon-revealer-${{ matrix.os }}/
```

Comment:
The executable flag setting is also missing here, I just re-added it in https://github.com/dungeon-revealer/dungeon-revealer/pull/427

Reply:
Whoops, forgot that.

Reply:
See c861f8b